demo/demo.jsx

```diff
@@ -105,14 +116,7 @@ export default class App extends React.Component {
         />
 
         <VictoryPie
-          data={_.range(0, 6)}
-          x={null}
```

Comment:
https://github.com/FormidableLabs/victory-pie/pull/65 has a fix to smooth this example out.